src/Gelf/Transport/IgnoreErrorTransportWrapper.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Gelf\Transport;
+
+use Gelf\MessageInterface as Message;
+
+/**
+ * A wrapper for any AbstractTransport to ignore any kind of errors
+ * @package Gelf\Transport
+ */
+class IgnoreErrorTransportWrapper extends AbstractTransport
+{
+
+    /**
+     * @var AbstractTransport
+     */
+    private $transport;
+
+    /**
+     * IgnoreErrorTransportWrapper constructor.
+     *
+     * @param AbstractTransport $transport
+     */
+    public function __construct(AbstractTransport $transport)
+    {
+        $this->transport = $transport;
+    }
+
+    /**
+     * Sends a Message over this transport.
+     *
+     * @param Message $message
+     *
+     * @return int the number of bytes sent
+     */
+    public function send(Message $message)
+    {
+        try {
+            return $this->transport->send($message);
+        } catch (\Exception $e) {
+            return 0;
```

Comment:
Sure, i could add this. What is about the naming, is IgnoreErrorTransportWrapper expressive enough?